CONTRIBUTING.md

```diff
@@ -101,3 +101,74 @@ This simply ensures that you have the legal right to make the contribution.
 
 Please feel free to add yourself to the CONTRIBUTORS.md file with your pull request.
 We appreciate your efforts and would love to add you to our list of contributors.
+
+## Tooling
+
+We use linters to standardize and maintain style in our files. 
+These linters are added as GitHub Actions to verify that each pull request is compatible.
+In this section, we provide instructions to detail how you can run these linters locally.
+
+In addition, we also describe the steps to deconstruct the rules to help their adoption for other use cases.
+
+### yamllint
+
+Validating the errors in your YAML files
+
+- install [yamllint](https://github.com/adrienverge/yamllint)
+- launch `yamllint`
+
+```shell
+yamllint .
+```
+
+### markdownlint
+
+Validating and fixing the errors in your Markdown files
+
+- install [markdownlint](https://github.com/markdownlint/markdownlint)
+- launch `markdownlint`
+
+```shell
+markdownlint --fix .
+```
+
```

Comment:
```suggestion

You can also use the [VS Code extension](https://github.com/DavidAnson/vscode-markdownlint) based on the [markdownlint library](https://github.com/DavidAnson/markdownlint) for Node.js.

```

We could add this here to let people know about the VS Code extension

Reply:
Good point